package.json

```diff
@@ -103,6 +103,7 @@
     "check:tsd": "npm run build:dts && tsd",
     "check:web": "WEB=true mocha test/node",
     "check:web-no-bigint": "WEB=true NO_BIGINT=true mocha test/node",
+    "check:bench":"npx ts-node test/bench/index.ts",
```

Comment:
Should we compile it to javascript and run that directly or is this fine?

Reply:
We should compile to JS, the overhead ts-node introduces is unknown and I don't think reflects real typical usage. 
Maybe a shell script or &&-ed commands to compile / clean files?